include/mesh/mesh_base.h

```diff
@@ -1016,13 +1016,17 @@ class MeshBase : public ParallelObject
    *  4.) call \p cache_elem_dims()
    *
    * The argument to skip renumbering is now deprecated - to prevent a
-   * mesh from being renumbered, set allow_renumbering(false).
+   * mesh from being renumbered, set allow_renumbering(false). The argument to skip
+   * finding neighbors is also deprecated. To prevent find_neighbors, set
+   * allow_find_neighbors(false)
    *
    * If this is a distributed mesh, local copies of remote elements
    * will be deleted here - to keep those elements replicated during
    * preparation, set allow_remote_element_removal(false).
    */
-  void prepare_for_use (const bool skip_renumber_nodes_and_elements=false, const bool skip_find_neighbors=false);
+  void prepare_for_use (const bool skip_renumber_nodes_and_elements, const bool skip_find_neighbors);
+  void prepare_for_use (const bool skip_renumber_nodes_and_elelments);
+  void prepare_for_use ();
```

Comment:
OK, so if I understand correctly, calling `prepare_for_use()` with no args is equivalent to allowing *both* renumbering and finding neighbors, i.e. the previous default values? I think I am good with that. It should be backwards compatible with all possible existing ways of calling `prepare_for_use()`, yes?

Reply:
Calling with no args will respect whatever the current values of `_skip_renumbering_nodes_and_elements` and `_skip_find_neighbors` are. If no one has touched these since normal mesh construction (e.g. not copy construction, since I'm going to try making the change you suggested), then these values are false, so we *allow* both.